jetty-util/src/main/java/org/eclipse/jetty/util/ssl/KeyStoreScanner.java

```diff
@@ -88,11 +78,23 @@ public KeyStoreScanner(SslContextFactory sslContextFactory, boolean followLinks)
         addBean(_scanner);
     }
 
+    private Path getRealKeyStorePath()
+    {
+        try
+        {
+            return keystoreFile.toPath().toRealPath();
+        }
+        catch (IOException e)
+        {
+            return keystoreFile.toPath();
+        }
+    }
+
     @Override
     public void fileAdded(String filename)
     {
         if (LOG.isDebugEnabled())
-            LOG.debug("added {}", filename);
+            LOG.debug("fileAdded {} - keystoreFile.toReal {}", filename, getRealKeyStorePath());
```

Comment:
I don't think we need to do a `toRealPath()` for debugging when we don't do it for the monitored file or for the scanner.

Reply:
it helps, in debugging, to see that it picked up the right file.